content/de/guide/v8/api-reference.md

```diff
@@ -1,13 +1,12 @@
 ---
-name: API-Referenzierung
-permalink: '/guide/api-reference'
+title: API-Referenzierung
 ---
 
 # API-Referenzierung
 
 ---
 
-<div><toc></toc></div>
+<toc></toc>
```

Comment:
I have a comment explaining this in `plugins/precompile-markdown/index.js`, but the gist of it is that the CommonMark spec defines which precise HTML elments can be used as a starting point for an HTML Block, and of course, something custom like `<toc>` doesn't fit the bill.

Without the wrapping `<div>` (as it is a valid starting identifier), `marked` would wrap the content in `<p>` which is not ideal. So instead, I added a little custom renderer to support `<toc>` here -- not essential, but removes an entirely superfluous node that serves an unclear purpose for everyone not familiar with CommonMark (which is probably everyone).